Userland/Services/KeyboardPreferenceLoader/main.cpp

```diff
@@ -30,12 +31,7 @@ ErrorOr<int> serenity_main(Main::Arguments)
     if (keymaps_vector.size() == 0)
         exit(1);
 
-    pid_t child_pid;
-    char const* argv[] = { "/bin/keymap", "-m", keymaps_vector.first().characters(), nullptr };
-    if ((errno = posix_spawn(&child_pid, "/bin/keymap", nullptr, nullptr, const_cast<char**>(argv), environ))) {
-        perror("posix_spawn");
-        exit(1);
-    }
+    TRY(Core::Process::spawn("/bin/keymap"sv, Array { "-m", keymaps_vector.first().characters() }));
```

Comment:
This creates a difference in behavior, since the original code did not disown the child process. That is important since the keymap binary should be forced to exit when the service exits. You can probably solve this with a `KeepAsChild` "enum boolean" argument.

Reply:
I'll add this anyway though since I think it's a handy option (and allows porting more places elsewhere).